### ISSUE TITLE
Deprecation with Twig 2.7

### ISSUE THREAD

**Issue by enumag** [liip/LiipImagineBundle#1166]
The spaceless tag used [here](https://github.com/liip/LiipImagineBundle/blob/master/Resources/views/Form/form_div_layout.html.twig#L2) is deprecated in Twig 2.7 and should be replaced by `{% filter spaceless %}`. This would however break compatibility with older Twig versions. Not sure how to fix it correctly.

**Comment by mikemix** [comment]
Don't use spaceless at all.

Instead of:

```
{% block liip_imagine_image_widget %}
    {% spaceless %}
        {% if image_path %}
            <div>
                {% if link_url %}
                    <a href="{{ link_filter ? link_url|imagine_filter(link_filter): link_url }}" {% for attrname, attrvalue in link_attr %}{{ attrname }}="{{ attrvalue }}" {% endfor %}>
                {% endif %}

                <img src="{{ image_path|imagine_filter(image_filter) }}" {% for attrname, attrvalue in image_attr %}{{ attrname }}="{{ attrvalue }}" {% endfor %} />

                {% if link_url %}
                    </a>
                {% endif %}
            </div>
        {% endif %}

        {{ block('form_widget_simple') }}
    {% endspaceless %}
{% endblock %}
```

use 

```
{% block liip_imagine_image_widget %}
        {%- if image_path -%}
            <div>
                {%- if link_url -%}
                    <a href="{{ link_filter ? link_url|imagine_filter(link_filter): link_url }}" {% for attrname, attrvalue in link_attr %}{{ attrname }}="{{ attrvalue }}" {% endfor %}>
                {%- endif -%}

                <img src="{{ image_path|imagine_filter(image_filter) }}" {% for attrname, attrvalue in image_attr %}{{ attrname }}="{{ attrvalue }}" {% endfor %} />

                {%- if link_url -%}
                    </a>
                {%- endif -%}
            </div>
        {%- endif -%}

        {{ block('form_widget_simple') -}}
{% endblock %}
```

**Comment by enumag** [comment]
@mikemix looks like you made the PR against the wrong repository.

**Comment by mikemix** [comment]
Lol you're right!

**Comment by dbu** [comment]
fixed in #1176
